samples/fullduplex/Core_8/Shared/AzureAuditBodyStorageConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Shared
+{
+    using Azure.Storage.Blobs;
+    using System;
+    using System.Threading.Tasks;
+
+    public static class AzureAuditBodyStorageConfiguration
+    {
+        public static async Task<BlobContainerClient> GetContainerClient()
+        {
+            var blobClient = new BlobServiceClient(Environment.GetEnvironmentVariable("AzureBlobAuditStorage"));
+
+            var blobContainerClient = blobClient.GetBlobContainerClient("audit-bodies");
```

Comment:
Maybe not worth it for the demo, but if we were to move forward with this, the container name will need to be configured exactly the same on both sides, or, if we think there would possibly be multiple containers, we will need to send it along in a header or something so ServiceControl looks in the right place for the body data.

Reply:
yea, I just hacked this together